developers/bindings/thing-xml.md

```diff
@@ -85,6 +85,7 @@ The following XML snippet shows a thing type definition with 2 channels and one
     <channels>
         <channel id="switch" typeId="powerSwitch" />
         <channel id="temperature" typeId="setpointTemperature" />
+        <channel id="roomHumidity" typeId="humidity" />
```

Comment:
Naming convention:
```suggestion
        <channel id="room-humidity" typeId="humidity" />
```

If you want to extend scope slightly, you could also correct the two channel type ids to comply with the now established naming convention.

Reply:
Umm. On this page, there are several more cases of either channel id or channel type id being camel case. So to be fully consistent I think we should change all instances of both channel id and channel type id from camelCase to dash-separated .. or??

Reply:
Yes, that would be even better. We don't want to encourage using camelCase only to be corrected when providing a PR.